<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PIDParametersUpdaterPanel</class>
 <widget class="QWidget" name="PIDParametersUpdaterPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Parameter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" rowspan="3">
    <widget class="QStackedWidget" name="panel_container">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>475</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QRadioButton" name="beginner_radio_button">
       <property name="text">
        <string>Beginner</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="intermediate_radio_button">
       <property name="text">
        <string>Intermediate</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="advance_radio_button">
       <property name="text">
        <string>Advance</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="default_button">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="pid_type_list">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
